cardano-node/src/Cardano/Tracing/OrphanInstances/Shelley.hs

```diff
@@ -317,9 +317,6 @@ instance Era era => ToObject (UtxowPredicateFailure era) where
              , "txBodyMetaDataHash" .= txBodyMetaDataHash
              , "fullMetaDataHash" .= fullMetaDataHash
              ]
-  toObject _verb InvalidMetaData =
-    mkObject [ "kind" .= String "InvalidMetaData"
-             ]
```

Comment:
Why are we removing this case? It still appears to be a constructor of the `UtxowPredicateFailure era` type.

Reply:
Not sure, the constructor is gone altogether.